gitlab/README.md

```diff
@@ -12,6 +12,12 @@ For more in-depth monitoring of your GitLab pipelines, check out [CI Pipeline Vi
 
 ## Setup
 
+This OpenMetrics-based integration has a latest mode (`use_openmetrics`: true) and a legacy mode (`use_openmetrics`: false). To get all the most up-to-date features, Datadog recommends enabling the latest mode. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][18].
+
+Metrics marked as `[OpenMetricsV1]` or `[OpenMetricsV2]` are only available using the corresponding mode of the GitLab integration. All other metrics are collected by both modes. 
+
+To use Prometheus/OpenMetrics v1 instead of OpenMetrics v2, change the `use_openmetrics` option to `use_prometheus`, and change the `openmetrics_endpoint` option to `prometheus_url`. For more information, see the [Prometheus and OpenMetrics metrics collection from a host documentation][30].
```

Comment:
This is related to my other comment. It's fine if we leave this information as a separate paragraph (though I think it should go right after the explanation about "modes"), but we should replace the references from v1 and v2 to legacy and latest like I've suggested in a different PR with a similar paragraph.

Reply:
```suggestion
To use the legacy mode of Prometheus/OpenMetrics instead of the latest mode of OpenMetrics, change the `use_openmetrics` option to `use_prometheus`, and change the `openmetrics_endpoint` option to `prometheus_url`. For more information, see the [Prometheus and OpenMetrics metrics collection from a host documentation][30].
```